osu.Game/Overlays/Mods/ModSelectScreen.cs

```diff
@@ -341,6 +358,75 @@ protected override void Update()
             }
         }
 
+        private class ColumnDimContainer : Container
+        {
+            public ModColumn Column { get; }
+
+            private readonly ScrollContainer<Drawable> parentScroll;
+            private readonly Bindable<bool> isFullyOnScreen = new BindableBool(true);
+
+            [Resolved]
+            private OsuColour colours { get; set; } = null!;
+
+            public ColumnDimContainer(ScrollContainer<Drawable> parentScroll, ModColumn column)
```

Comment:
I'd definitely prefer a `RequestScroll` flow rather than passing down the whole scroll container.